php/Terminus/CommandWithSSH.php

```diff
@@ -31,6 +31,29 @@ protected function formatOutput($string) {
   }
 
   /**
+   * Parses server information for connections
+   *
+   * @param [array]  $site_info  Elements as follows:
+   *        [string] site        Site UUID
+   *        [string] environment Environment name
+   * @return [array] $server Connection info
+   */
+  protected function getServerInfo(array $site_info = array()) {
```

Comment:
It's a protected function, but I think we might be specific that this gets an Application Server. Just to avoid confusion about what services are where. Especially cause this has port 2222 hardcoded, which means that it's "connection to ssh into an app server", rather than other kinds of servers.